prime-dotnet-webapi/Models/EnrolmentCertificate.cs

```diff
@@ -35,9 +36,17 @@ public static EnrolmentCertificate Create(Enrollee enrollee)
                 PreferredLastName = enrollee.PreferredLastName,
                 DateOfBirth = enrollee.DateOfBirth,
                 LicensePlate = enrollee.LicensePlate,
-                Privileges = enrollee.AssignedPrivileges.Select(ap => ap.Privilege),
                 OrganizationTypes = enrollee.Organizations.Select(org => org.OrganizationType),
-                EnrolmentCertificateNote = enrollee.EnrolmentCertificateNote
+                EnrolmentCertificateNote = enrollee.EnrolmentCertificateNote,
+                Transactions = enrollee.AssignedPrivileges
+                                .Select(ap => ap.Privilege)
+                                .Where(p => p.PrivilegeGroup.PrivilegeTypeCode == 2),
+                UserType = enrollee.AssignedPrivileges
+                            .Select(ap => ap.Privilege)
+                            .Where(p => p.PrivilegeGroupCode == 4).FirstOrDefault(),
+                CanHaveOBOs = enrollee.AssignedPrivileges
+                            .Select(ap => ap.Privilege)
+                            .Where(p => p.PrivilegeGroupCode == 5).FirstOrDefault(),
```

Comment:
Don't really like the magic numbers here, but its probably fine.
There isn't an obvious ordering for these last two, I assume SingleOrDefault() is more appropriate.

Reply:
K changed it to SingleorDefault. Agree about the 'magic numbers'. Do you have a suggestion for a better way?

Reply:
Prime Constant?

Reply:
You could do for example what we've done with the status reasons with some named constants on the model. That's still not a flawless solution since those have to be maintained if the data changes but it at least names the magic numbers.

Reply:
k I made that change!